docs/anomaly-detection/docs/models/models.md

```diff
@@ -0,0 +1,325 @@
+
+*Config Example*
+<div class="with-copy" markdown="1">
+
+```yaml
+model:
+  class: "model.mad.MADModel"
+  threshold: 2.5
+```
+Resulting metrics of the model are described [here](#vmanomaly-output).
+
+---
+## [Z-score](https://en.wikipedia.org/wiki/Standard_score)
+*Parameters specific for vmanomaly*:
+\* - mandatory parameters.
+* `class`\* (string) - model class name `"model.zscore.ZscoreModel"`
+* `z_threshold` (float) - [standard score](https://en.wikipedia.org/wiki/Standard_score) for calculation boundaries and anomaly score. Defaults to 2.5.
+
+*Config Example*
+<div class="with-copy" markdown="1">
+
+```yaml
+model:
+  class: "model.zscore.ZscoreModel"
+  z_threshold: 2.5
+```
+</div>
+
+Resulting metrics of the model are described [here](#vmanomaly-output).
+
+## [Isolation forest](https://en.wikipedia.org/wiki/Isolation_forest) (Multivariate)
+Detects anomalies using binary trees. The algorithm has a linear time complexity and a low memory requirement, which works well with high-volume data. It can be used on both univatiate and multivariate data, but it is more effective in multivariate case.
+
+**Important**: Be aware of [the curse of dimensionality](https://en.wikipedia.org/wiki/Curse_of_dimensionality). Don't use single multivariate model if you expect your queries to return many time series of less datapoints that the number of metrics. In such case it is hard for a model to learn meaningful dependencies from too sparse data hypercube.
+
+Here we use Isolation Forest implementation from `scikit-learn` [library](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.IsolationForest.html). All parameters from this library can be passed to the model.
+
+*Parameters specific for vmanomaly*:
+
+\* - mandatory parameters.
+* `class`\* (string) - model class name `"model.isolation_forest.IsolationForestMultivariateModel"`
+
+* `contamination` - The amount of contamination of the data set, i.e. the proportion of outliers in the data set. Used when fitting to define the threshold on the scores of the samples. Default value - "auto". Should be either `"auto"` or be in the range (0.0, 0.5].
+
+* `args`: (dict) - Inner model args (key-value pairs). See accepted params in [model documentation](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.IsolationForest.html). Defaults to empty (not provided). Example:  {"random_state": 42, "n_estimators": 100}
+
+*Config Example*
+<div class="with-copy" markdown="1">
+
+```yaml
+model:
+  # To use univariate model, substitute class argument with "model.isolation_forest.IsolationForestModel".
+  class: "model.isolation_forest.IsolationForestMultivariateModel"
+  contamination: "auto"
+  args:
+    n_estimators: 100
+    # i.e. to assure reproducibility of produced results each time model is fit on the same input
+    random_state: 42
+```
+</div>
+
+Resulting metrics of the model are described [here](#vmanomaly-output).
+
+---
+## Custom model
```

Comment:
My suggestion is to add custom model guide right in this place or into Guides section.

Reply:
agreed to intentionally leave it as is in initial version, see comments above, so we have cross-references and do not have oversized pages with many nested subsection levels